packages/web-wallet/README.md

```diff
@@ -0,0 +1,139 @@
+# Hathor Web Wallet
+
+Web interface for interacting with Hathor blockchain via MetaMask Snap.
+
+## Overview
+
+Browser-based wallet that connects to the Hathor network through a MetaMask Snap extension. Supports HTR and custom token transfers, transaction history, NFT detection, and network switching between mainnet and testnet.
+
+## Prerequisites
+
+- Node.js 22+
+- Yarn 4.x
+- MetaMask browser extension with Snap support
+- Running Hathor Snap (local development or installed)
+
+## Quick Start
+
+```bash
+# Install dependencies (from monorepo root)
+yarn install
+
+# Start development server
+yarn workspace @hathor/web-wallet dev
+
+# Access at http://localhost:5173
```

Comment:
Tried and MetaMask said loading local snaps is disabled, can we also have a small segment on any configuration that we need to do on the MetaMask to make it work?

Reply:
Done in 3418382f839a42bcc6abd82664bce1142232f411